src/rsz/include/rsz/Resizer.hh

```diff
@@ -739,6 +749,9 @@ class Resizer : public dbStaState
 
   dpl::Opendp* opendp_ = nullptr;
 
+  std::map<Corner*, std::ofstream> spef_file;
+  bool write_spef_file;
```

Comment:
Give a default value, eg `bool write_spef_file = false;`.   Is the necessary or is it equivalent to spef_files.empty()?